Initial Commit

cseydlitz opened the pull request:
Adding all files for lab partner to check
@michaelGRU  please review

cseydlitz committed:
Initial Commit

Adding all files for lab partner to check

michaelGRU commented:
great intro. im so proud of u!! theres some huge gaps of white space between some paragraphs in the pdf version. maybe try to fix it? we should also discuss how to interpret some of the histograms.

cseydlitz committed:
Proposed final upload

Did further research; described some previously unknown features in the given data. This allowed me to make a more educated deduction about what data I would use to optimize a background.

cseydlitz commented:
Sorry @michaelGRU I'm unable to fix the whitespace in the PDF, that's just how it is because of my graphs and such. 

@oliviatessa Please review
